mapclassify/classifiers.py

```diff
@@ -809,6 +840,104 @@ def find_bin(self, x):
             right[right == len(self.bins)] = len(self.bins) - 1
         return right
 
+    def plot(
+        self,
+        gdf,
+        border_color="lightgrey",
+        border_width=0.10,
+        title=None,
+        legend=False,
+        cmap="YlGnBu",
+        axis_on=True,
+        legend_kwds={"loc": "lower right"},
```

Comment:
I know `splot` is inconsistent on using [`kwds`](https://github.com/pysal/splot/search?q=kwds&unscoped_q=kwds) versus [`kws`](https://github.com/pysal/splot/search?q=kws&unscoped_q=kws) for keyword arguments, but [`seaborn` uses `kws`](https://seaborn.pydata.org/generated/seaborn.regplot.html#seaborn.regplot), and that makes more sense to me? Why the `d`?

Reply:
`kwds` came from following [geopandas](https://stackoverflow.com/questions/53225923/geopandas-legend-kwds-seems-to-be-ignored).

Since `Mapclassify.plot` is wrapping gpd, I thought it made sense to go with that?

Reply:
gotcha, makes sense! We should pick one and stick with it going forward... no big deal either way, but we should shoot for consistency going forward. 

matplotlib uses just `_kw` so it's far from standardized across this space I guess.

Reply:
good catch, I think this should be changed in splot too. I like both `_kw` and `_kws`, shorter is nicer.